### ISSUE TITLE
Windows 7 Protocol Mismatch

### ISSUE THREAD

**Issue by Firewall999** [brave/brave-browser#8494]
## Description 

Windows allows bookmark links to be attached to the Brave browser icon in the taskbar so users may elect to open Brave and go directly to the bookmark selected with a right click on the Brave taskbar icon.  This does not work with Version 1.4.95 Chromium: 80.0.3987.122 (Official Build) (64-bit).  The Error message reported is shown in the attached file:  
![Brave Protocol Bug Error Message_01](https://user-images.githubusercontent.com/44746065/75795245-0c11ba80-5d40-11ea-8b70-d6fbf5ed2942.PNG)

## Steps to Reproduce

  1.  In Brave go to any normal, simple site, such as yahoo.com, and pin Brave to the taskbar.
  2.  Left-click in the address bar to activate it and select the whole address.
  3.  Left-click and hold over the address, move it over to the Brave icon in the taskbar, and drop it.
  4.  Right-click on the Brave icon in the taskbar and left-click on the link you just put there.
  5.  The error message should read, "Unable to open this shortcut.  The protocol 'BraveHTML' does not have a registered program."

## Actual result:
![Brave Protocol Bug Error Message_01](https://user-images.githubusercontent.com/44746065/75796394-9575bc80-5d41-11ea-8200-e149147385a4.PNG)

## Expected result:
Should open the link with no error message.

## Reproduces how often: 
Always

## Brave version (brave://version info)
Version 1.4.95 Chromium: 80.0.3987.122 (Official Build) (64-bit)

## Version/Channel Information:
Windows 7 version

- Can you reproduce this issue with the current release?  Yes
- Can you reproduce this issue with the beta channel?  N/A
- Can you reproduce this issue with the dev channel?  N/A
- Can you reproduce this issue with the nightly channel?  N/A

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields?  Unknown
- Does the issue resolve itself when disabling Brave Rewards?  Unknown
- Is the issue reproducible on the latest version of Chrome?   Unknown

## Miscellaneous Information:
There are indeed html protocols registered for bookmarks.  They are just not the same ones that the taskbar is looking for.  I suspect that somewhere in the code they are inconsistently named.

**Comment by bsclifton** [comment]
@Firewall999 did you use the older version of Brave? Handlers for that may still be registered- if you still have that installed (version would be 0.26 or earlier) please uninstall it and the registry should get cleaned up

**Comment by Firewall999** [comment]
No, I did not have a version of Brave that is 0.26 or earlier.  Nothing but the current version to uninstall.  My current version is Version 1.9.72 Chromium: 81.0.4044.138 (Official Build) (64-bit).

**Comment by rebron** [comment]
@Firewall999 We've had quite a few chromium updates since this was last reported. Are you still having this issue with our latest 1.15.72?

**Comment by bsclifton** [comment]
This issue hasn't had an update in a while - it's met the criteria for being stale. The issue is going to be closed for now. If this is still a problem, please leave a comment or re-open the issue. Thanks!
